modules/gapi/test/infer/gapi_infer_ov_tests.cpp

```diff
@@ -0,0 +1,492 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2023 Intel Corporation
+
+#ifdef HAVE_INF_ENGINE
+
+#include "../test_precomp.hpp"
+
+#include "backends/ov/util.hpp"
+
+#include <opencv2/gapi/infer/ov.hpp>
+
+#include <openvino/openvino.hpp>
+
+namespace opencv_test
+{
+
+namespace {
+// FIXME: taken from DNN module
+void initDLDTDataPath()
+{
+#ifndef WINRT
+    static bool initialized = false;
+    if (!initialized)
+    {
+        const char* omzDataPath = getenv("OPENCV_OPEN_MODEL_ZOO_DATA_PATH");
+        if (omzDataPath)
+            cvtest::addDataSearchPath(omzDataPath);
+        const char* dnnDataPath = getenv("OPENCV_DNN_TEST_DATA_PATH");
+        if (dnnDataPath) {
+            // Add the dnnDataPath itself - G-API is using some images there directly
+            cvtest::addDataSearchPath(dnnDataPath);
+            cvtest::addDataSearchPath(dnnDataPath + std::string("/omz_intel_models"));
+        }
+        initialized = true;
+    }
+#endif // WINRT
+}
+
+#if INF_ENGINE_RELEASE >= 2020010000
+static const std::string SUBDIR = "intel/age-gender-recognition-retail-0013/FP32/";
+#else
+static const std::string SUBDIR = "Retail/object_attributes/age_gender/dldt/";
+#endif
```

Comment:
Do you still need the old dir layout or we just need to update CI / opencv_extra?

Reply:
Yes, as far as I know layout remains the same

Reply:
Remains the same where? We don't need the `#else` branch today, right?